src/ui/SubPageSendAdvanced.qml

```diff
@@ -11,6 +13,63 @@ import "Dialogs/" // For quick UI development, switch back to resources when mak
 
 Page {
     id: subPageSendAdvanced
+    function getSelectedAddresses(){
```

Comment:
I think that this function is wrong because you iterate over the items of the popup, there are not all the items only those that are rendered at that time, i think a better approach would be return the array used for cach the checked state